website/docs/docs/deploy/job-commands.md

```diff
@@ -49,9 +48,78 @@ You can add or remove as many dbt commands as necessary for every job. However,
 Use [selectors](/reference/node-selection/syntax) as a powerful way to select and execute portions of your project in a job run. For example, to run tests for one_specific_model, use the selector: `dbt test --select one_specific_model`. The job will still run if a selector doesn't match any models. 
 
 :::
-   
-    
-**Job outcome**  &mdash;  During a job run, the commands are "chained" together and executed as run steps. If one of the run steps in the chain fails, then the subsequent steps aren't executed, and the job will fail.
+
+#### Test metrics in CI <Lifecycle status="team,enterprise" />
```

Comment:
is it just ent/team given SL is for ent/teams?

Reply:
Yes for right now, we plan to enable developer creds for the SL soon.